src/Utils/Hooks/useMedia.ts

```diff
@@ -26,6 +26,8 @@ export function useMedia(): { [k in Breakpoint]?: boolean } {
     (acc, [key, value]) => {
       return {
         ...acc,
+        // TODO:
+        // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
OK So this guy here. It looks like it's just being used in a single component, which I can just rewrite to use the useMatchMedia hook and then remove this one (it doesn't look like it's used in force?) Otherwise we should refactor these two a bit.

Reply:
If its only being used once feel free to remove 👍